packages/react-sdk/src/tree/root.ts

```diff
@@ -23,8 +23,12 @@ export type Data = {
   params?: Params;
 };
 
+export type RootPropsData = Omit<Data, "build"> & {
+  build: Pick<Data["build"], "instances" | "props">;
+};
```

Comment:
Could it be `Omit<Data, "build"> & { instances: Build['instances'], props: Build['props'] }`?

Reply:
In the future definitely yes, for now it just same type as used at saas